kobuki_gazebo_plugins/src/gazebo_ros_kobuki_loads.cpp

```diff
@@ -309,6 +309,8 @@ void GazeboRosKobuki::setupRosApi(std::string& model_name)
 {
   std::string base_prefix;
   gazebo_ros_->node()->param("base_prefix", base_prefix, std::string("mobile_base"));
```

Comment:
If the parameter server has the empty string as the value of `"base_prefix"`, then `ros::NodeHandle::param()` will use it and not the default. The [API description of param()](http://docs.ros.org/jade/api/roscpp/html/classros_1_1NodeHandle.html#af0bf7baa0ab2385275bb657cc358111b) claims that the default value is only used if the parameter value cannot be retrieved.